src/main/webapp/userdir.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Student Recommendations</title>
```

Comment:
nit: maybe add a more meaningful title (as this is the general site, not page specific)